src/app/server/Server.cpp

```diff
@@ -330,7 +330,8 @@ CHIP_ERROR Server::Init(const ServerInitParams & initParams)
 #endif // CHIP_CONFIG_ENABLE_ICD_SERVER
 
 #if INET_CONFIG_ENABLE_TCP_ENDPOINT
-    app::DnssdServer::Instance().SetTCPServerEnabled(mTransports.GetTransport().GetImplAtIndex<1>().IsServerListenEnabled());
+    // Enable the TCP Server based on the TCPListenParameters setting in TransportMgr Init.
```

Comment:
Please put the TcpListenParameters on the stack above, so that here we can just use the boolean from those and keep the two things in sync that way.

Reply:
Ok. Can do that. Kept it as-is thinking would make that change when we up-level the TcpListenParameters to be set from higher up.